### DISCUSSION TITLE
add a slugify tool

### DISCUSSION THREAD

**Discussion by wc-matteo** [CorentinTh/it-tools#267] [Ideas]
it would create a url safe path from a string. It should convert accented letters to non-accented ones (e.g. è => e). Like this: https://www.npmjs.com/package/@sindresorhus/slugify

**Comment by CorentinTh** [comment]
Done !
It's live : https://it-tools.tech/slugify-string

![screely-1676064228393](https://user-images.githubusercontent.com/25065347/218200157-8a5b8a5f-6dc0-4070-b011-f5e88c9f2a07.png)

**Comment by wc-matteo** [comment]
amazing! It'd be nice if you could choose the separator character as well .-.

**Comment by sharevb** [comment]
Hi @CorentinTh and @wc-matteo, separator implemented in PR #1232

**Reply by sharevb** [reply]
Hi, if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
